src/main/java/domain/money/Money.java

```diff
@@ -0,0 +1,24 @@
+package domain.money;
+
+public class Money {
+    private final int value;
+
+    public Money(final int value) {
+        this.value = value;
+    }
+
+    public int getValue() {
+        return value;
+    }
+
+    public Money multiply(double multipleValue) {
+        Double resultValue = value * multipleValue;
+        return new Money(resultValue.intValue());
+    }
+
+    @Override
+    public boolean equals(Object obj) {
+        Money other = (Money) obj;
+        return value == other.value;
+    }
```

Comment:
'equals()' should check the class of its parameter 라는 권고가 있어요 
```suggestion
        @Override
    public boolean equals(Object o) {
        if (this == o) return true;
        if (o == null || getClass() != o.getClass()) return false;
        Money money = (Money) o;
        return value == money.value;
    }
```

Reply:
그리고 equals와 hashCode는 함께 재정의하길 권장해요!

https://tecoble.techcourse.co.kr/post/2020-07-29-equals-and-hashCode/

Reply:
감사합니다!!